### PR TITLE
Fix imagerunner saucectl config file

### PR DESCRIPTION
**Pull request by FriggaHel**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/orchestrate/saucectl-configuration.md [hunk 1]
```diff
@@ -36,14 +36,150 @@ Tells `saucectl` this is a Sauce Orchestrate configuration. `imagerunner` is the
...
+  region: eu-central-1
+```
+
+---
+
+### `concurrency`
+
+<p><small>| OPTIONAL | INTEGER |</small></p>
+
+Sets the maximum number of suites to execute at the same time. If the test defines more suites than the max, excess suites are queued and run in order as each suite completes.
```
Comment (aliciakrebs) [inline docs/orchestrate/saucectl-configuration.md:71]:
```suggestion
Sets the maximum number of suites to run at the same time. If the test defines more suites than the max, excess suites are queued and run in order as each suite completes.
```

**Review by aliciakrebs — APPROVED** [review]

File: docs/orchestrate/saucectl-configuration.md [hunk 2]
```diff
@@ -36,14 +36,150 @@ Tells `saucectl` this is a Sauce Orchestrate configuration. `imagerunner` is the
...
+---
+
+### `region`
+
+<p><small>| OPTIONAL | STRING/ENUM |</small></p>
+
+Specifies through which Sauce Labs data center tests will run. Valid values are: `us-west-1` or `eu-central-1`.
+
+```yaml
+  region: eu-central-1
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:62]: I think it's helpful to see the parent context:

```suggestion
sauce:
  region: eu-central-1
```

File: docs/orchestrate/saucectl-configuration.md [hunk 3]
```diff
@@ -36,14 +36,150 @@ Tells `saucectl` this is a Sauce Orchestrate configuration. `imagerunner` is the
...
+---
+
+### `concurrency`
+
+<p><small>| OPTIONAL | INTEGER |</small></p>
+
+Sets the maximum number of suites to run at the same time. If the test defines more suites than the max, excess suites are queued and run in order as each suite completes.
+
+```yaml
+  concurrency: 3
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:74]:
```suggestion
sauce:
  concurrency: 3
```

Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:74]:
```suggestion
sauce:
  concurrency: 3
```

File: docs/orchestrate/saucectl-configuration.md [hunk 4]
```diff
@@ -36,14 +36,150 @@ Tells `saucectl` this is a Sauce Orchestrate configuration. `imagerunner` is the
...
+---
+
+### `retries`
+
+<p><small>| OPTIONAL | INTEGER |</small></p>
+
+Sets the number of times to retry a failed suite. For more settings, you can refer to [passThreshold](#passThreshold).
+
+```yaml
+  retries: 1
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:92]:
```suggestion
sauce:
  retries: 1
```

File: docs/orchestrate/saucectl-configuration.md [hunk 5]
```diff
@@ -135,6 +271,77 @@ The following limitations are in effect for artifact downloads. They do not appl
...
 :::
 
+### `cleanup`
+
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+When set to `true`, all contents of the specified download directory are cleared before any new artifacts from the current test are downloaded.
+
+```yaml
+  cleanup: true
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:281]:
```suggestion
artifacts:
  cleanup: true
```

File: docs/orchestrate/saucectl-configuration.md [hunk 6]
```diff
@@ -135,6 +271,77 @@ The following limitations are in effect for artifact downloads. They do not appl
...
+---
+
+### `download`
+
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+Specifies the settings related to downloading artifacts from tests run by `saucectl`.
+
+```yaml
+  download:
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:293]:
```suggestion
artifacts:
  download:
```

File: docs/orchestrate/saucectl-configuration.md [hunk 7]
```diff
@@ -135,6 +271,77 @@ The following limitations are in effect for artifact downloads. They do not appl
...
+
+Specifies when and under what circumstances to download artifacts. Valid values are:
+
+- `always`: Always download artifacts.
+- `never`: Never download artifacts.
+- `pass`: Download artifacts for passing suites only.
+- `fail`: Download artifacts for failed suites only.
+
+```yaml
+    when: always
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:314]:
```suggestion
artifacts:
  download:
    when: always
```

File: docs/orchestrate/saucectl-configuration.md [hunk 8]
```diff
@@ -135,6 +271,77 @@ The following limitations are in effect for artifact downloads. They do not appl
...
+---
+
+#### `match`
+
+<p><small>| OPTIONAL | STRING/ARRAY |</small></p>
+
+Specifies which artifacts to download based on whether they match the name or file type pattern provided. Supports the wildcard character `*` (use quotes for best parsing results with wildcard).
+
+```yaml
+  match:
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:326]:
```suggestion
artifacts:
  download:
    match:
```

File: docs/orchestrate/saucectl-configuration.md [hunk 9]
```diff
@@ -135,6 +271,77 @@ The following limitations are in effect for artifact downloads. They do not appl
...
+---
+
+#### `directory`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Specifies the path to the folder location in which to download artifacts. A separate subdirectory is generated in this location for each suite for which artifacts are downloaded. The name of the subdirectory will match the suite name. If a directory with the same name already exists, the new one will be suffixed by a serial number.
+
+```yaml
+    directory: ./artifacts/
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:340]:
```suggestion
artifacts:
  download:
    directory: ./artifacts/
```

File: docs/orchestrate/saucectl-configuration.md [hunk 10]
```diff
@@ -36,14 +36,154 @@ Tells `saucectl` this is a Sauce Orchestrate configuration. `imagerunner` is the
...
+---
+
+### `concurrency`
+
+<p><small>| OPTIONAL | INTEGER |</small></p>
+
+Sets the maximum number of suites to run at the same time. If the test defines more suites than the max, excess suites are queued and run in order as each suite completes.
+
+```yaml
+sauce:
```
Comment (alexplischke) [inline docs/orchestrate/saucectl-configuration.md:75]: One too many :D
```suggestion
```

**Review by alexplischke — APPROVED** [review]

**Pull request merged**